src/analyses/constant_propagator.cpp

```diff
@@ -218,8 +214,11 @@ bool constant_propagator_domaint::two_way_propagate_rec(
 #ifdef DEBUG
   std::cout << "two_way_propagate_rec: " << from_expr(ns, "", expr) << '\n';
 #endif
-  bool change = false;
 
+  bool change=false;
+
+  // this seems to be buggy at present
+#if 0
```

Comment:
If it is broken, shouldn't it be removed entirely?

Reply:
I'd like to keep it for now if that's OK.  One of the patched disabled this but doesn't remove it as it should be fix-able but was giving unsound results.  Although I think we should move to using the variable sensitivity domain, I get that some people are still using this and may wish to continue to do so.  This allows them to fix it if they want.